integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -72,6 +71,37 @@ const getSafariCapabilities = capabilities => {
   return cap
 };
 
+const getChromeOptions = capabilities => {
+  let chromeOptions = {};
+  if (capabilities.useFakeMedia) {
+    chromeOptions['args'] = ['--use-fake-device-for-media-stream', '--use-fake-ui-for-media-stream'];
+    if (capabilities.video) {
+      chromeOptions['args'].push('--use-file-for-fake-video-capture=' + fetchMediaPath(capabilities.video, capabilities.browserName));
+    }
+    if (capabilities.audio) {
+      chromeOptions['args'].push('--use-file-for-fake-audio-capture=' + fetchMediaPath(capabilities.audio, capabilities.browserName));
+    }
+  }
+  return chromeOptions;
```

Comment:
Do we need to remove following chromeOptions?
'start-maximized',
      'disable-infobars',
      'ignore-gpu-blacklist',
      'test-type',
      'disable-gpu',
 
We had recently added them based on SauceLabs recommendation, they said our tests were missing these default options hence we added them. Could not find if we are adding them elsewhere, hence, checking once.